backend/server/routers/movies.py

```diff
@@ -22,6 +28,44 @@
 @router.get("/movies")
 async def movies(user_id: str = Depends(auth)):
     results = await DB.query(
-        "SELECT `id`, `title`, `release_date` as `showings_start`, `poster_url` FROM `MovieShowing`"
+        "SELECT id, title, release_date as showings_start, poster_url FROM MovieShowing"
     )
     return results
+
+
+
+@router.post("/movie/review", status_code=200)
+async def create_movie_review(
+    review: CreateMovieReviewRequest,
+    user_id: str = Depends(auth),
+):
+    # Check if the user has already reviewed the movie
+    checkForReview = await DB.queryOne(
+        "SELECT * FROM `Review` WHERE `user` = :user_id AND `movie` = :movie_id",
+        {"user_id": user_id, "movie_id": review.movie_id},
+    )
+    
+    # If a review already exists, return a conflict error
+    if checkForReview is not None:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail="You have already reviewed this movie."
+        )
+    
+    # Generate a unique ID for the new review
+    new_review_id = newId("short")  # Use your newId function to generate a unique review ID
```

Comment:
```suggestion
    new_review_id = newId()  # Use your newId function to generate a unique review ID
```
This will need to be a standard ID, not a short one, but other than that it looks good.